outlines/processors/base_logits_processor.py

```diff
@@ -129,6 +145,11 @@ def _from_torch(tensor: torch.Tensor, target_type: Type) -> Array:
             # numpy doesn't support bfloat16, mlx doesn't support direct conversion from torch
             return mx.array(tensor.float().numpy())
 
+        elif is_jax_array_type(target_type):
+            import jax
+
+            return jax.numpy.from_dlpack(tensor)
```

Comment:
I'm wondering if we need to use `jax.numpy`? Is there a reason to not just use `jax.dlpack.from_dlpack`?

Reply:
you are right, we can just use `jax.dlpack.from_dlpack` , idk why autocomplete suggestions didn't show it, and I came across links that used `jax.numpy.dlpack`, though I will check whats difference it creates really